.eslintignore

```diff
@@ -1 +1,2 @@
-integration/side-effects/snapshots/
\ No newline at end of file
+integration/side-effects/snapshots/
+docs_app/src/ie-polyfills.js
```

Comment:
Please remove this change so that configuration/build environment changes are not included with documentation changes. AFAICT, ESLint is only run over `.js` files that are included in commits and there are no such files in this PR.

Reply:
This is kind of interesting because I did not changed this file but when I rebased my branch onto master and pushed, this change appeared. I also checked the git log on this file and I do not have any commit on it.

Reply:
I think this is just part in the stable branch, we might want to exclude eslint for the docs app completly. So for now I would appreciate if you could revert this change, so that we can have a proper decision how to proceed with eslint and the docs app

Reply:
OK thanks, I reverted the eslint related change.